packages/apm/src/integrations/tracing.ts

```diff
@@ -505,6 +514,16 @@ export class Tracing implements Integration {
 
     Tracing._log('[Tracing] Adding & adjusting spans using Performance API');
 
+    // FIXME: depending on the 'op' directly is brittle.
+    if (transactionSpan.op === 'pageload') {
+      // Force any pending records to be dispatched.
+      Tracing._forceLCP();
+      if (Tracing._lcp) {
+        // Set the last observed LCP score.
+        transactionSpan.setData('_sentry_extra_metrics', { lcp: Tracing._lcp });
```

Comment:
@HazAT @kamilogorek @dashed PTAL

The idea here is to have a "namespace" where we can add other browser metrics later.

Using `Transaction.data` for now, but we can brainstorm how we want to do this going forward.

Reply:
@rhcarvalho I'm open to have a meeting to talk more about this "namespace" to store these kinds of metrics.